native/Cargo.toml

```diff
@@ -21,11 +21,6 @@ safe-api = "0.5.0"
 neon = "0.3.1"
```

Comment:
were we not wanting to update this to 0.3.2 ?

Reply:
yes, and I thought it was upgraded since the lock file had a ref to new version, I'll take a look at this, since I'm seeing an error in node v10 when changing it due to some limitations in stdout buffer